### DISCUSSION TITLE
Allow users to signup

### DISCUSSION THREAD

**Discussion by Monarsoft** [immich-app/immich#9925] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Allow user registration, you can check whether to open it. If necessary, you can check to allow users to register, so that there are more choices.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
The built in auth is deliberately very minimal. If you want features like signup, you can use oauth.
